.github/workflows/build-and-publish-chart.yml

```diff
@@ -50,7 +54,6 @@ jobs:
       - name: Build Dependencies
```

Comment:
Line 40-49 above assume we are merging from develop to main... which means any beta functionality is merged into main. Do we really want that? Vs trying to make this work from a feature-XXXX branch?

Reply:
I'll second this. Any subsequent merges to `main` that yield a stable version will include the "beta" changes from a prior pre-release version, which isn't ideal.

Reply:
Added a condition that only merges when the user specifies to do so: https://github.com/Cloudzero/cloudzero-charts/pull/84/commits/c85a8c373295c39654544bac2c59234b69561dd8

Reply:
@bdrennz looking over this file - I think there are other locations which are impacted (ie. gh-pages). Would we want to maybe make a `clone` of this workflow focused only on feature branches?

Reply:
> Would we want to maybe make a clone of this workflow focused only on feature branches?

Ya I think that would be the best option, we can always consolidate later.


> looking over this file - I think there are other locations which are impacted (ie. gh-pages).

I think it's worth considering some of the benefits I mention in my [comment](https://github.com/Cloudzero/cloudzero-charts/pull/84#issuecomment-2417854590). But if you have strong opinions against, I don't mind removing that step.